### DISCUSSION TITLE
GH not detecting diff between branches

### DISCUSSION THREAD

**Discussion by LLLisa** [community/community#39385] [Pull Requests]
### Select Topic Area

Bug

### Body

GH is not telling me there is no difference between two branches, and therefore I cannot make a PR. However, when I open code under those two branches I can visually confirm there is a difference.

**Comment by samus-aran** [comment]
Hi @LLLisa 👋 Welcome to the Community. 

How did you get on, are you still having this issue? I often make the mistake of not checking the base: and compare: if that helps 😅

**Reply by LLLisa** [reply]
The issue is still there, but we found a workaround. By adding a comment to the files we know are different, GH shows us all of the differences in the online editor, so we were able to resolve it that way. I asked some other folks and they said that they'd never seen or heard of it before. The guy that made the initial merge said it's likely that the mistake was his, but at this point it's a mystery.

**Reply by samus-aran** [reply]
Ack that's strange! 🤔 Glad you got it sorted though 👍

**Reply by LLLisa** [reply]
Thank you for checking!
